/**:
  ros__parameters:
    voxel_size: 10.0
    pose_graph_subsampling_factor: 10
    visualization_update_period_ms: 1000
    enable_keypoints_visualization: false
    enable_pointclouds_visualization: true
    pose_graph_markers_size: 1.0
    produce_mesh: false
    nb_colors: 10
